src/csync/vio/csync_vio_local_win.cpp

```diff
@@ -158,8 +161,12 @@ std::unique_ptr<csync_file_stat_t> csync_vio_local_readdir(csync_vio_handle_t *d
           return nullptr;
       }
   }
+  auto path = c_utf8_from_locale(handle->ffd.cFileName);
+  if (path == "." || path == "..")
```

Comment:
You could change that check in `csync_ftw` to an assert now that it's taken care of here. Or will the server send us such file entries?

Reply:
I don't think the server send .. or . ,  but maybe it does.